### PR TITLE
Add polygon Aurem & Dew nft models

### PR DESCRIPTION
**Pull request by BennyFeng**
# Thank you for contributing to Spellbook!
Please refer to the top of the `readme` in the root of Spellbook to learn how to contribute to Spellbook on DuneSQL.

### CODE REVIEW

**Comment by BennyFeng** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by BennyFeng** [comment]
@0xRobin The price in `enrich_nft_trades` macro only supports `ethereum`. I didn't change it, but I think it will get an incorrect price on other chains. Do you have any thoughts on this?

**Comment by 0xRobin** [comment]
> @0xRobin The price in `enrich_nft_trades` macro only supports `ethereum`.

@BennyFeng this is something we missed! I'll fix asap in another PR.

**Comment by BennyFeng** [comment]
@0xRobin @jeff-dude @Hosuke Could you please review this PR asap? This polygon PR is quite urgent.

File: models/_sector/nft/trades/polygon/nft_polygon_base_trades.sql [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        platform_fee_address,
+        royalty_fee_address,
+        sub_tx_trade_id,
+        tx_from,
+        tx_to,
+        tx_data_marker
+    FROM {{ nft_model }}
+    {% if is_incremental() %}
+    WHERE {{incremental_predicate('block_time')}}
+    {% endif %}
```
Comment (0xRobin) [inline models/_sector/nft/trades/polygon/nft_polygon_base_trades.sql:46]:
```suggestion
```
This can be removed since it's a view.

**Review by 0xRobin — APPROVED** [review]

**Pull request merged**